exotic.py

```diff
@@ -288,20 +242,26 @@ def new_getParams(data):
     planetDictionary = {
         'pName': data['pl_name'],
         'sName': data['hostname'],
-        'rprs': data['pl_radj']*rjup/(data['st_rad']*rsun),
-        'aRs': data['pl_ratdor'],
-        'midT': data['pl_tranmid'],
-        'midTUnc': data['pl_tranmiderr1'],
+        'ra': data['ra'],
+        'dec': data['dec'],
         'pPer': data['pl_orbper'],
         'pPerUnc': data['pl_orbpererr1'],
-        'flag': data['tran_flag'],
-        'inc': data["pl_orbincl"],
-        'ecc': data.get("pl_orbeccen",0),
-        'teff': data["st_teff"],
-        'met': data["st_met"],
-        'logg': data["st_logg"],
-        'ra': data["ra"],
-        'dec': data["dec"]
+        'midT': data['pl_tranmid'],
+        'midTUnc': data['pl_tranmiderr1'],
+        'rprs': data['pl_radj']*rjup/(data['st_rad']*rsun),
+        'aRs': data['pl_ratdor'],
+        'inc': data['pl_orbincl'],
+        'ecc': data.get('pl_orbeccen', 0),
+        'teff': data['st_teff'],
+        'teffUncPos': data['st_tefferr1'],
+        'teffUncNeg': data['st_tefferr2'],
+        'met': data['st_met'],
+        'metUncPos': data['st_meterr1'],
+        'metUncNeg': data['st_meterr2'],
+        'logg': data['st_logg'],
+        'loggUncPos': data['st_loggerr1'],
+        'loggUncNeg': data['st_loggerr2'],
+        'flag': data['tran_flag']
```

Comment:
Scraping both the + and - uncertainties for temperature, surface gravity, and metallicity.